lib/util.js

```diff
@@ -6,11 +6,14 @@ var appPort = parseInt(argv.mocking_port, 10);
 // +1 is used for selennium
 var mocksPort = appPort + 2;
 
+var mocksHttpsPort = appPort + 3;
```

Comment:
I don't believe there is a +3 available (I think the next nightwatch worker will take that port).  AKA, there are 3 ports available per nightwatch process.  Dave C can tell you for sure though.

Do you actually need both http and https starting up or just https?